reset-pass/account-validator-tool/src/bundle/org/sakaiproject/accountvalidator/bundle/messages.properties

```diff
@@ -10,7 +10,7 @@ msg.resetPassword=Reset your {0} password.
 validate.title=Account Validation
 validate.invalidPassword=Password is incorrect
 validate.passNotMatch=Passwords don't match
-validate.userNotDefined=Your account does not exist
+validate.userNotDefined=This user account no longer exists in {0}. Please create/request a new account.
```

Comment:
How about just: Please request instead of Please create/request

Reply:
No problem.